agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/view/AgendaEventAttendeeItem.vue

```diff
@@ -62,6 +77,9 @@ export default {
     isAttendeeSpace() {
       return this.attendee && this.attendee.identity && this.attendee.identity.providerId === 'space';
     },
+    isAttendeeGuest() {
+      return this.attendee && this.attendee.identity && this.attendee.identity.providerId === 'AGENDA_GUEST_USER';
```

Comment:
```suggestion
      return this.attendee && this.attendee.identity && this.attendee.identity.providerId === 'GUEST_USER';
```
To change default provider name to make it generic.

Reply:
Done